.gitignore

```diff
@@ -108,4 +111,4 @@ oprofile_data/
 /plugins/zerospu2/Windows/dsound51.cpp
 /tools/bin
 /tools/GSDumpGUI/bin
-/tools/GSDumpGUI/obj
+/tools/GSDumpGUI/obj
```

Comment:
Can you readd the newline (so it looks like you haven't touched the end of the file)? I can merge after that's done.